verifiable/verifiable.cu

```diff
@@ -157,6 +179,22 @@ struct ReduceSum {
       return hip_bfloat16(static_cast<float>(a) + static_cast<float>(b));
   }
   #endif
+  #if RCCL_FLOAT8 == 1
+  __host__ __device__ rocblas_f8 operator()(rocblas_f8 a, rocblas_f8 b) const {
+    #if __CUDA_ARCH__ >= 800
```

Comment:
We need to replace this to ROCm flag

Reply:
Also, seems rccl-tests always convert to 32-bit for half/bfloat. It doesn't do native half/bfloat reduce operation.

Reply:
updated.